sleap/gui/app.py

```diff
@@ -812,7 +814,31 @@ def new_instance_menu_action():
 
         ### Tracks Menu ###
```

Comment:
This should be now labeled as:
```suggestion
        ### Sessions Menu ###
```